Documentation/data-format.md

```diff
@@ -0,0 +1,69 @@
+# Adobe Experience Platform Edge Bridge for iOS
+
+## Data Format
```

Comment:
how about simply Data format?
Also don't forget to add it to toc -> https://github.com/adobe/aepsdk-edgebridge-ios/blob/main/Documentation/README.md